src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java

```diff
@@ -360,14 +357,14 @@ public Map<MavenProject, Xml.Document> parseMaven(List<MavenProject> mavenProjec
             mavenExecutionContext.setPomCache(getPomCache(pomCacheDirectory, logger));
         }
 
-        List<String> activeProfiles = topLevelProject.getActiveProfiles().stream().map(Profile::getId).collect(Collectors.toList());
+        List<String> activeProfiles = topLevelProject.getActiveProfiles().stream().map(Profile::getId).collect(toList());
         if (!activeProfiles.isEmpty()) {
             mavenParserBuilder.activeProfiles(activeProfiles.toArray(new String[]{}));
         }
 
         List<Xml.Document> mavens = mavenParserBuilder
                 .build()
-                .parse(allPoms, baseDir, ctx).collect(toList());
+                .parse(allPoms, baseDir, ctx).collect(Collectors.toList());
```

Comment:
Here you're slightly inconsistent as compared to elsewhere in the same file; figured use the same everywhere.
```suggestion
                .parse(allPoms, baseDir, ctx).collect(toList());
```